rules/windows/defense_evasion_process_termination_followed_by_deletion.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/11/04"
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a process termination event quickly followed by the deletion of its executable file. Malware tools and other non-native
+files dropped or created on a system by an adversary may leave traces to indicate to what occurred. Removal of these files can occur during an intrusion, or as part of a post-intrusion process to minimize the
+adversary's footprint.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Process Termination followed by Deletion"
+risk_score = 47
+rule_id = "09443c92-46b3-45a4-8f25-383b028b258d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5s
+   [process where event.type == "end" and 
+    process.code_signature.trusted == false and
+    not process.executable : ("C:\\Windows\\SoftwareDistribution\\*.exe", "C:\\Windows\\WinSxS\\*.exe")
+   ] by process.executable
+   [file where event.type == "deletion" and file.extension : "exe"] by file.path
```

Comment:
I noticed there were some other extensions (SCR, COM) in the issue, were those meant to be excluded?

Reply:
forgot them, nice catch.